dev/WindowsAppRuntime_BootstrapDLL/MddBootstrap.cpp

```diff
@@ -131,16 +162,29 @@ STDAPI_(void) MddBootstrapShutdown() noexcept
         g_endTheLifetimeManagerEvent.reset();
     }
 
+    HRESULT hrLifetimeManagerShutdown = S_OK;
     if (g_lifetimeManager)
     {
-        (void)LOG_IF_FAILED(g_lifetimeManager->Shutdown());
+        hrLifetimeManagerShutdown = g_lifetimeManager->Shutdown();
+        (void)LOG_IF_FAILED(hrLifetimeManagerShutdown);
```

Comment:
Why this change?

Reply:
to capture the error, if any, in the Stop event down under. This is because Shutdown is best effort and we don't stop Shutdown activity on error.

Reply:
Ahhh. That wasn't clear. Then make it

`hrLifetimeManagerShutdown = LOG_IF_FAILED(g_lifetimeManager->Shutdown());`

`LOG_*()` return their parameter, they just happen to log as it passes through